specification.md

```diff
@@ -238,7 +238,7 @@ Required parameters
 
 - The **baggage key**, a string
 
-**Returns** the corresponding **baggage value**, xor some indication that such a value was missing.
+**Returns** the corresponding **baggage value**, or some indication that such a value was missing.
```

Comment:
`xor` might have been intentional here, indicating an ["exclusive or"](https://en.wikipedia.org/wiki/Exclusive_or). I don't think your changes cause any loss of meaning here, but an alternative might have been:

> **Returns** either the corresponding **baggage value**, or some indication that such a value was missing.

Reply:
Thank you for the valuable feedback @jpkrohling 
I shall revert it to 'xor', since it's more specific and unambiguous as compared to using 'either....or...'

Reply:
I personally prefer "either...or", as it gives the right meaning. I personally tend to dislike "xor" on documentation.

Reply:
Yay! I do too! :+1: Will change it right away.